### DISCUSSION TITLE
Leaderboard in regression

### DISCUSSION THREAD

**Discussion by talhaanwarch** [autogluon/autogluon#1222] [Q&A] [answered]
What does the leaderboard show in regression? I have defined eval_metric='root_mean_squared_error'. I am getting some negative values. 
Using `num_bag_folds=5` i got following score
train_set's rmse: 0.0003166	valid_set's rmse: 0.693966

**Comment by Innixma** [comment, marked as answer]
AutoGluon Tabular always shows scores in higher-is-better format. This means error values have their signs flipped so it is instead a score. An RMSE score_val of -5 is the same as an RMSE of 5.

**Reply by Innixma** [reply]
This is useful for a number of reasons, mostly because now the code a user can write to determine the relative ranking of models is independent of the metric being either a score metric or error metric, higher will always be better.

**Reply by talhaanwarch** [reply]
ok, thanks. it clear.
